src/main/java/com/brandwatch/internship/fetchandstoretweets/entities/Query.java

```diff
@@ -2,17 +2,30 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-
-import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
 
 public class Query {
 
-    private static AtomicLong counter = new AtomicLong();
+    @ResponseStatus(value=HttpStatus.NOT_FOUND, reason="No such Query")  // 404
```

Comment:
If we have this line, do we still need `ControllerAdvice`?  I think we only need one of them.
Also spaces are missing around `=`

Reply:
There is `ExceptionHanlderController.java` which is `@ControllerAdvice`

Reply:
Ah sorry, did not understand u well. Yes I needed one because it is the one that handles the exceptions, other than that, it redirects to /error which I do not have. I think I do not have to have this `ControllerAdvice` but I will need to make this /error page and render the error.